### PR TITLE
Docker - Add native libs

### PR DESCRIPTION
**Pull request by pmauduit**
This is still in a WIP state, do not merge for now.

### CODE REVIEW

File: mapfishapp/src/docker/Dockerfile [hunk 1]
```diff
@@ -4,8 +4,17 @@ ADD . /
...
+RUN ln -s /usr/share/java/gdal.jar /var/lib/jetty/lib/ext/
+
+
 VOLUME [ "/var/local/uploads" ]
 
 ENTRYPOINT [ "/docker-entrypoint.sh" ]
 CMD ["java","-Djava.io.tmpdir=/tmp/jetty", "-Dgeorchestra.datadir=/etc/georchestra","-jar","/usr/local/jetty/start.jar"]
 
+
+
```
Comment (fvanderbiest) [inline mapfishapp/src/docker/Dockerfile]: extra lines at EOF

**Comment by fvanderbiest** [comment]
This looks good, thanks.

**Comment by pmauduit** [comment]
I guess it can be safely merged, but waiting for @Vampouille opinion.

File: mapfishapp/src/docker/Dockerfile [hunk 2]
```diff
@@ -4,6 +4,13 @@ ADD . /
 
 RUN java -jar "$JETTY_HOME/start.jar" --add-to-startd=jmx,jmx-remote,stats
 
+RUN apt-get update && \
+   apt-get install -y libgdal1h libgdal-java gdal-bin && \
```
Comment (Vampouille) [inline mapfishapp/src/docker/Dockerfile]: `gdal-bin` depends on `libgdal1h` : https://packages.debian.org/jessie/gdal-bin
So you can remove `libgdal1h`

File: extractorapp/src/docker/Dockerfile [hunk 3]
```diff
@@ -4,6 +4,12 @@ ADD . /
 
 RUN java -jar "$JETTY_HOME/start.jar" --add-to-startd=jmx,jmx-remote,stats
 
+RUN apt-get update && \
+   apt-get install -y libgdal1h libgdal-java gdal-bin && \
```
Comment (Vampouille) [inline extractorapp/src/docker/Dockerfile]: `gdal-bin` depends on `libgdal1h`, so you can safely remove `libgdal1h` package here.

Comment (pmauduit) [inline extractorapp/src/docker/Dockerfile]: Yup

**Pull request merged**